### ISSUE TITLE
Suggestion: Add support for the Atom text editor

### ISSUE THREAD

**Issue by finnbear** [ide-stories/vscode-stories#93]
In the future, please consider adding support for other text editors. Atom would be a good first addition, because it is highly extensible.

**Comment by bar9** [comment]
Good idea. That would probably be a new repository that somebody needs to create. I could also imagine putting the different editors on product hunt and see which one gets the most votes. My personal favorite will be vim in any case :smile_cat:
